### PR TITLE
Updates - Room Attendant and User Entity.

### PR DESCRIPTION
**Pull request by sammu1991**
Changes to Room Attendant Backend and Front end and User Entity.

### CODE REVIEW

File: roomranger/src/main/java/org/launchcode/roomranger/models/RoomAttendant.java [hunk 1]
```diff
@@ -6,52 +6,50 @@
...
-
     private String workingDays ;
     @Column(unique = true)
     private String email;
     private String notes;
-    private int remainingDays = 20;
+
+
+
+    private int remainingDays;
```
Comment (LunaLiu829) [inline roomranger/src/main/java/org/launchcode/roomranger/models/RoomAttendant.java:41]: Please keep this initial value as 20. I used this for leave system

**Comment by sammu1991** [comment]
Can we hardcode in table directly?

File: roomranger/src/main/java/org/launchcode/roomranger/models/User.java [hunk 2]
```diff
@@ -2,57 +2,25 @@
 
 import jakarta.persistence.*;
 import java.util.Set;
-//CHANGED LONG ID TO INT ID
+
 @Entity
-public class User {
+public class User extends AbstractEntity{
```
Comment (LunaLiu829) [inline roomranger/src/main/java/org/launchcode/roomranger/models/User.java:7]: Sonal is also using this table, I am not sure it is fine if you change this class

**Comment by sammu1991** [comment]
I am trying to keep user table simple.. just username pwd role and id..

File: roomranger/src/main/java/org/launchcode/roomranger/models/User.java (hunk 2 above)
Comment (sammu1991) [reply roomranger/src/main/java/org/launchcode/roomranger/models/User.java:7]: Ok.. after Sonal's review I will change it

**Pull request closed without merging**